solidity/ecdsa/test/utils/operators.ts

```diff
@@ -18,55 +22,50 @@ export type Operator = {
 
 export async function registerOperators(
   walletRegistry: WalletRegistry,
-  tToken: T,
-  addresses: Address[],
+  t: T,
+  numberOfOperators: number,
+  unnamedSignersOffset = 0,
```

Comment:
Shouldn't it be 10 by default?

Reply:
🤷🏻 Yeah, we could do that.

Reply:
If we don't do that, then one of the operators/staking providers/beneficiaries/authorizers uses the same address as deployer, third party etc. Correct?

Reply:
It won't overlap with the deployer as it's a named account. We define offset based on the use case in tests, for example here:
https://github.com/keep-network/keep-core/blob/e691e2406f0be1ed4b89a52e9f7ae783cb49a7e0/solidity/ecdsa/test/fixtures/index.ts#L77-L96

We pull one unnamed signer for `thirdParty` hence we define offset as `1`.

Reply:
Ah, I see now. Let's leave it as-is then.

Reply:
Please see if https://github.com/keep-network/keep-core/pull/2910 helps anything.

Reply:
it does!